test/integration/artefact_router_registration_test.rb

```diff
@@ -70,10 +70,10 @@ def artefact_details_hash(attrs = {})
 
     should "register the correct prefix and exact paths with the router" do
       r2 = WebMock.stub_request(:put, "#{@router_api_base}/routes").
-        with(:body => {"route" => hash_including("incoming_path" => "/bar", "route_type" => "prefix")}).
+        with(:body => {"route" => hash_including("incoming_path" => "/bar")}).
         to_return(:status => 201)
       r3 = WebMock.stub_request(:put, "#{@router_api_base}/routes").
-        with(:body => {"route" => hash_including("incoming_path" => "/foo.json", "route_type" => "exact")}).
+        with(:body => {"route" => hash_including("incoming_path" => "/foo.json")}).
```

Comment:
These 2 changes don't look correct.  We should still be sending the route_type for `PUT` requests.  It's just the delete requests that no longer need them...